test/jobs_cache_test.rb

```diff
@@ -4,6 +4,7 @@ class JobsCacheTest < MiniTest::Test
   describe ConeyIsland::JobsCache do
     before do
       @instance = ConeyIsland::JobsCache.new
+      @instance.flush_jobs
```

Comment:
![God damn it](http://gifrific.com/wp-content/uploads/2012/07/ron-swanson-computer-throw-out-parks-and-rec.gif)